BoxContentSDK/BoxContentSDK/Requests/BOXFileRepresentationDownloadRequest.m

```diff
@@ -116,16 +116,19 @@ - (void)performRequestWithProgress:(BOXProgressBlock)progressBlock completion:(B
         }
         
         fileOperation.successBlock = ^(NSString *modelID, long long expectedTotalBytes) {
-            NSError *dataIntegrityErrorIfAny = nil;
-            if([self.sha1Hash length] && ![self.sha1Hash isEqualToString: [BOXHashHelper sha1HashOfFileAtPath:self.destinationPath]]) {
-                dataIntegrityErrorIfAny = [[NSError alloc] initWithDomain:BOXContentSDKErrorDomain code:BOXContentSDKDataIntegrityError userInfo:nil];
-                if(dataIntegrityErrorIfAny) {
-                    [[NSNotificationCenter defaultCenter] postNotificationName:BOXFileDownloadCorruptedNotification object:self];
-                }
-            }
             [BOXDispatchHelper callCompletionBlock:^{
-                completionBlock(dataIntegrityErrorIfAny);
+                completionBlock(nil);
             } onMainThread:isMainThread];
+
+            NSString *calculatedSha1 = [BOXHashHelper sha1HashOfFileAtPath:self.destinationPath];
```

Comment:
Are we only doing the sha1 check on the original item? Or does it check all representation types?

Reply:
We do the check if the string exists. However, currently, the BOX API only returns the sha1 for the original content.